acc_test/resource_kafka_test.go

```diff
@@ -7,13 +7,16 @@ import (
 	"os"
 	"regexp"
 	"strconv"
+
 	"testing"
 
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
 	"github.com/instaclustr/terraform-provider-instaclustr/instaclustr"
 )
 
+// we put the test here so that the Kafka cluster can be reused amongst bunch of other Kafka related tests
```

Comment:
Not sure what this comment means?
TestKafkaResource in in resource_kafaka_test.go. Seems like a reasonable place to me?

Reply:
ah right, what I meant there was that the Kafka Connect testing is bunched here even though logically it should be part of the resource_cluster_test

Reply:
ok, should probably mention Kafka Connect specifically if that's what its about